<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Framwork</class>
 <widget class="QWidget" name="Framwork">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>493</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Framwork</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="10,150,10">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="titleBarLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>30</number>
     </property>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="centerLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>400</number>
     </property>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="statusBarLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>30</number>
     </property>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
